<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RipCDDialog</class>
 <widget class="QDialog" name="RipCDDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="input_group">
     <property name="title">
      <string>Input options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Rip</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Track</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Duration</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="select_all_button">
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="select_none_button">
            <property name="text">
             <string>Select None</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="invert_selection_button">
            <property name="text">
             <string>Invert Selection</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="2" column="3" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="year_label">
          <property name="text">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="disc_label">
          <property name="text">
           <string>Disc</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="album_label">
          <property name="text">
           <string>Album</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="6">
         <widget class="QLineEdit" name="albumLineEdit"/>
        </item>
        <item row="2" column="6">
         <widget class="QLineEdit" name="discLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="4" alignment="Qt::AlignLeft">
         <widget class="QLineEdit" name="yearLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="artist_label">
          <property name="text">
           <string>Artist</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="genreLineEdit"/>
        </item>
        <item row="2" column="0" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="genre_label">
          <property name="text">
           <string>Genre</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="6">
         <widget class="QLineEdit" name="artistLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="output_group">
     <property name="title">
      <string>Output options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QPushButton" name="select">
        <property name="text">
         <string>Select...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>File Format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="format_filename">
        <property name="text">
         <string notr="true">%tracknum% - %artist% - %track%</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Destination</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="format">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="destination">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="options">
        <property name="text">
         <string>Options...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Audio format</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="progress_group">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QProgressBar" name="progress_bar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>select_all_button</tabstop>
  <tabstop>select_none_button</tabstop>
  <tabstop>invert_selection_button</tabstop>
  <tabstop>albumLineEdit</tabstop>
  <tabstop>artistLineEdit</tabstop>
  <tabstop>genreLineEdit</tabstop>
  <tabstop>yearLineEdit</tabstop>
  <tabstop>discLineEdit</tabstop>
  <tabstop>format_filename</tabstop>
  <tabstop>format</tabstop>
  <tabstop>options</tabstop>
  <tabstop>destination</tabstop>
  <tabstop>select</tabstop>
  <tabstop>button_box</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
